catalog/task/add-bundles-to-image-index/0.1/add-bundles-to-image-index.yaml

```diff
@@ -0,0 +1,112 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: add-bundles-to-image-index
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+      Submit a build request to add operator bundles to an index image
+  params:
+    - name: IIB_URL 
+      description: IIB service url
+      default: "https://iib.engineering.redhat.com/api/v1"
+      type: string
+    - name: IIB_JSON_BUILD_ARGS
+      description: IIB JSON arguments to add a new build
+      type: string
+    - name: IIB_SECRET
+      type: string
+      default: "iib-service-secret"
+      description: Secret with iib credentials to be used
+    - name: IIB_BUILD_TIMEOUT_SECONDS
+      type: string
+      default: "300"
+      description:  Timeout seconds to receive the build state
+  results:
+    - name: json_build_info
+      description: JSON build information for the requested build
+    - name: build_state
+      description: IIB Service build state
+  steps:
+    - name: add-bundles-to-image-index
+      image: quay.io/hacbs-release/release-utils:latest
```

Comment:
I'm not sure how best to handle this, but I don't think this should be latest. I looked, and it looks like we push this image on every commit and tag it with the commit. So ideally, this would be pointing to the commit that this commit generates once merged. Maybe it can be done in a followup PR? I don't think it should be latest because in a year we can change the image and remove something this task needs and then version 0.1 of the task is broken. I believe that no task or pipeline should ever use `main` or `latest` tags

Reply:
I agree. Although we need to change the building process then. Where do we do that? I did not see any other tag than `lastest`  at `hacbs-release/release-utils`. Maybe my user has some permission missing? (not even sure that quay.io has such kind of view permission) or we might need to change the building process to reflect your suggestion.

Reply:
https://github.com/hacbs-release/release-bundles/blob/main/.github/workflows/dockerfile_push.yaml#L32 Maybe we haven't updated this repo since the workflow was added? I also only see the latest tag for the image right now